src/main/java/org/example/Homework16.java

```diff
@@ -24,3 +29,5 @@ public static void LoginEmptyEmailPasswordTest()  {
         driver.quit();
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void loginEmptyEmailPasswordTest ()`

![image](https://user-images.githubusercontent.com/104761482/209428163-6311d74d-be0e-481c-8572-f70201317e43.png)